---
bool success
float32 panMin
float32 panMax
float32 tiltMin
float32 tiltMax